gobblin-utility/src/main/java/gobblin/util/Either.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2014-2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.util;
+
+import lombok.AllArgsConstructor;
+
+import com.google.common.base.Optional;
+
+
+/**
+ * Created by ibuenros on 10/29/15.
```

Comment:
I assume you just forgot about this due to the large amount of code changes :-).